CHANGELOG.unreleased.md

```diff
@@ -8,6 +8,7 @@
 > Users must be able to say: “Nice enhancement, I'm eager to test it”
 
 - [VM Creation] Automatically create a VTPM if the template requests it (Windows templates starting from XCP-ng 8.3) (PR [#7436](https://github.com/vatesfr/xen-orchestra/pull/7436))
+- [Pool/Advanced] Default SR can now be configured from the pool advanced tab [#7414](https://github.com/vatesfr/xen-orchestra/issues/7414) (PR [#7451](https://github.com/vatesfr/xen-orchestra/pull/7451))
```

Comment:
Use formatting to make it easier to identify special words, for instance you can use `code` for error messages, code or filenames, *italic* for names appearing verbatim in the UI (and write them exactly as they are, like respecting capitalization) or feature names (e.g. *Incremental Replication*). These tools are at our disposal to make the text easier to understand for readers :slightly_smiling_face: 

```suggestion
- [Pool/Advanced] Default SR can now be configured from the pool's *Advanced* tab [#7414](https://github.com/vatesfr/xen-orchestra/issues/7414) (PR [#7451](https://github.com/vatesfr/xen-orchestra/pull/7451))
```